### PR TITLE
Feature/role page

### PR DESCRIPTION
**Pull request by braulioportela79**
Role page for the user choose between Freelancer or Job Owner. This page is accessed by the user after activation by email.

rolepage1

rolepage2

rolepage3

### CODE REVIEW

File: libs/components/src/lib/role-selection-form/role-selection-form.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { StyledButton } from "@freelance/components";
+import { Form, RadioGroup } from "./role-selection-form.styled";
+
+export function RoleSelectionForm() {
+	const { t } = useTranslation();
+	const navigate = useNavigate();
+	const [radioOption, setRadioOption] = useState("");
```
Comment (lyudad) [inline libs/components/src/lib/role-selection-form/role-selection-form.tsx]: can we set null?
and you forget to add type

Comment (braulioportela79) [reply libs/components/src/lib/role-selection-form/role-selection-form.tsx]: fixed type

Comment (braulioportela79) [reply libs/components/src/lib/role-selection-form/role-selection-form.tsx]: fixed type

Comment (braulioportela79) [reply libs/components/src/lib/role-selection-form/role-selection-form.tsx]: fixed set null

File: libs/components/src/lib/form-container/form-container.tsx [hunk 2]
```diff
@@ -46,6 +47,7 @@ export function FormContainer({
 			{formType === "login" && <LoginForm />}
 			{formType === "signup" && <SignupForm />}
 			{formType === "recoverPass1" && <RecoverPassFirstForm />}
+			{formType === "roleSelection" && <RoleSelectionForm />}
```
Comment (lyudad) [inline libs/components/src/lib/form-container/form-container.tsx]: Снимок экрана 2022-11-28 в 20 31 25

Comment (braulioportela79) [reply libs/components/src/lib/form-container/form-container.tsx]: Fixed.

**Review by lyudad — APPROVED** [review]

**Pull request merged**